.github/workflows/push.yml

```diff
@@ -38,11 +38,6 @@ jobs:
       run: python3 setup.py build
     - name: Install
       run: python3 setup.py install
-    - name: Test
-      run: |
-        cd tests
-        JUNIT_REPORT=1 ./all.sh
```

Comment:
Instead of removing it entirely I would rather have this replaced with the pytest part.